adobe/react-spectrum
# fix(#3220): DragManager force forms mode when dragging

Closes #3220 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding React Spectrum GitHub Issue #3220. 

## 📝 Test Instructions:

1. Using Windows with either JAWS or NVDA screen reader, open https://reactspectrum.blob.core.windows.net/reactspectrum/686489b2774e1238e3c4f1abc42593cbbba5e92b/storybook/index.html?path=/story/drag-and-drop--default&id=drag-and-drop--default&providerSwitcher-toastPosition=bottom&viewMode=story
2. In browse mode/virtual cursor mode, navigate to the button labelled "Drag me." 
3. **The `dragDescriptionVirtual` message, "Click to start dragging." does not get announced.**
4. Press Space or Enter key to activate the "Drag me" button and start dragging.
5. NVDA or JAWS should shift into forms mode.
6. The live announcer will announce the, `dragStartedVirtual` message, "Started dragging. Navigate to a drop target, then click or press Enter to drop," followed by the `endDragVirtual` message, "Dragging. Click to cancel drag."
7. Use down arrow to move the virtual cursor to the first available drop target, a button labelled, "Drop here."
8. The screen reader should to announce the `dropDescriptionVirtual`, "Click to drop."
9. Press Enter key to complete drop action.
10. The live announcer will announce the `dropComplete` message, "Drop complete."

## 🧢 Your Project:

Adobe/Accessibility
